src/mlx/warnings.py

```diff
@@ -39,12 +42,23 @@ def return_doxygen_warnings(self):
         print("{count} doxygen warnings found".format(count=self.doxygen_counter))
         return self.doxygen_counter
 
+    def check_junit_failures(self, line):
+        '''
+        Function for counting the number of JUnit warnings in a xmlfile.
+        The function returns the number of failing test cases found
+        '''
+        self.junit_counter += len(re.findall(junit_pattern, line))
```

Comment:
yes, this only is a counter. I didn't print the warnings/failures as the 2 other ones also don't print their  warnings. This makes sense as the parsing is from a terminal output which usually appears right on before the warnings plugin is invoked, so in CI you'll see one on top of the other. For JUnit we're parsing xml which is not displayed in the terminal.

Maybe we could add a feature request, where we have a command argument to generically enable/disable output detailed warning information per case ?